.gitignore

```diff
@@ -54,6 +54,48 @@ doc/man1/openssl-ts.pod
 doc/man1/openssl-verify.pod
 doc/man1/openssl-x509.pod
 
+# Auto generated doc files
```

Comment:
How about merging this with the list above? ☝🏽

Reply:
rebase goof.  merged and fixup commit pushed; thanks.